Common/src/main/java/me/egg82/antivpn/apis/vpn/IPInfo.java

```diff
@@ -0,0 +1,86 @@
+package me.egg82.antivpn.apis.vpn;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import me.egg82.antivpn.APIException;
+import me.egg82.antivpn.utils.ValidationUtil;
+import ninja.egg82.json.JSONWebUtil;
+import ninja.leaping.configurate.ConfigurationNode;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class IPInfo extends AbstractSourceAPI {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private static AtomicInteger hourlyRequests = new AtomicInteger(0);
+    private static AtomicInteger minuteRequests = new AtomicInteger(0);
```

Comment:
GetIPIntel was the one with a limit of 15 requests/min.
According to their API: `If you are using the API interface, please do not exceed more than 500 queries per day & 15 queries per minute. Custom packages are available if you contact me. More information is available in the FAQs.`
https://getipintel.net/free-proxy-vpn-tor-detection-api/#assumptions